addons/l10n_co/models/res_partner.py

```diff
@@ -0,0 +1,24 @@
+# coding: utf-8
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.depends('vat')
+    def _compute_verification_code(self):
+        for partner in self.filtered('vat'):
+            factors = zip(partner.vat, [71, 67, 59, 53, 47, 43, 41, 37, 29, 23, 19, 17, 13, 7, 3][15 - len(partner.vat):])
```

Comment:
Yeah, runbot told me to not use map (and zip I think). So I ended up rewriting the function because it was a bit more complicated than it had to be anyway. By the way, it's kind of weird, this was fixed somewhere last week. I guess GitHub saves the review and doesn't say anything when the code has changed?